firmware/user/modes/mode_flight.h

```diff
@@ -0,0 +1,13 @@
+/*
+ * mode_flappy.h
```

Comment:
Nice copy/paste (not that I care at all)

Reply:
Fixing